site/en/blog/shared-element-transitions-for-spas/index.md

```diff
@@ -161,7 +162,7 @@ The elements don't need to be the same node. In this case, `element2` will trans
 
 **Shared transition elements will render on top of other elements.** If other elements render on top of the transitioned elements, they'll pop underneath for the transition. We're not sure yet if this should be a general limitation, or if it should be fixed.
 
-**Shared transition elements must have `contain: paint`.** This makes it difficult to transition elements that have paint effects outside their content rectangle, such as shadows, blurs, and outlines. Hopefully this will change in future.
+**Shared transition elements must have [`contain: paint`](/content-visibility/#containment).** This makes it difficult to transition elements that have paint effects outside their content rectangle, such as shadows, blurs, and outlines. Hopefully this will change in future.
```

Comment:
```suggestion
**Shared transition elements must have [`contain: paint`](https://web.dev/content-visibility/#containment).** This makes it difficult to transition elements that have paint effects outside their content rectangle, such as shadows, blurs, and outlines. Hopefully this will change in future.
```